cardano-cli/test/cli/transaction-sign/run

```diff
@@ -36,9 +36,10 @@ assert_file_exists "${TEST_DIR}/signed.tx"
 assert_string 'TxSignedShelley' "${TEST_DIR}/signed.tx"
 
 # Sign a pool registration transaction.
+#TODO: This needs to use an unsigned tx with a registration certificate
```

Comment:
@dcoutts I'll be implementing the certificate golden tests in hedgehog anyways. I can come back and fix this rather than doing the same work twice.